doc/api/esm.md

```diff
@@ -667,10 +667,16 @@ imported from the same path.
 
 ## Wasm modules
 
+<!-- YAML
+changes:
+  - version: REPLACEME
+    pr-url: https://github.com/nodejs/node/pull/57038
+    description: Wasm modules no longer require the `--experimental-wasm-modules` flag.
+-->
+
 > Stability: 1 - Experimental
```

Comment:
```suggestion
> Stability: 1.2 - Release candidate
```

Reply:
I've marked this as 1.1 - Active development still for now.